Bottom: 2b7e02c4135fa3d5ec0db28c5fce13ba86ca6b0d
Top:    b4e14070b1eb5e20167e14bbf82f43a506666aa2
Author: Lawrence D'Anna <lawrence_danna@apple.com>
Date:   2019-10-03 11:08:43 -0700

Refresh of allow-arbitrary-python-streams

---
diff --git a/lldb/packages/Python/lldbsuite/test/python_api/file_handle/TestFileHandle.py b/lldb/packages/Python/lldbsuite/test/python_api/file_handle/TestFileHandle.py
index 306bb9b33a6..c33101ee162 100644
--- a/lldb/packages/Python/lldbsuite/test/python_api/file_handle/TestFileHandle.py
+++ b/lldb/packages/Python/lldbsuite/test/python_api/file_handle/TestFileHandle.py
@@ -455,8 +455,7 @@ class FileHandleTestCase(lldbtest.TestBase):
         with open(self.out_filename, 'w') as f:
             sbf = lldb.SBFile(f)
             status = self.debugger.SetOutputFile(sbf)
-            if status.Fail():
-                raise Exception(status)
+            self.assertTrue(status.Success())
             self.handleCmd('script 2+2')
         with open(self.out_filename, 'r') as f:
             self.assertEqual(f.read().strip(), '4')
@@ -467,8 +466,7 @@ class FileHandleTestCase(lldbtest.TestBase):
         with open(self.out_filename, 'w') as f:
             sbf = lldb.SBFile(f)
             status = self.debugger.SetErrorFile(sbf)
-            if status.Fail():
-                raise Exception(status)
+            self.assertTrue(status.Success())
             self.handleCmd('lolwut', check=False, collect_result=False)
         with open(self.out_filename, 'r') as f:
             errors = f.read()
